chain/chain/src/update_shard.rs

```diff
@@ -70,10 +70,6 @@ pub enum ShardUpdateReason {
 /// Information about shard to update.
 pub struct ShardContext {
     pub shard_uid: ShardUId,
-    /// Whether node cares about shard in this epoch.
-    pub cares_about_shard_this_epoch: bool,
```

Comment:
Seems like we don't care about shard this epoch any more, lol! Was this and `will_shard_layout_change` not being used anywhere?

Reply:
It was part of resharding v2 logic.